train.py

```diff
@@ -113,9 +113,14 @@ def makeLossBatch(outputs, batch, attns, range_):
         """Create all the variables that need to be sharded.
         This needs to match compute loss exactly.
         """
+        if opt.copy_attn:
+            # Only copy attn uses alignment
+            align = batch.alignment[range_[0] + 1: range_[1]]
+        else:
+            align = None
         return {"out": outputs,
                 "target": batch.tgt[range_[0] + 1: range_[1]],
-                "align": batch.alignment[range_[0] + 1: range_[1]],
+                "align": align,
```

Comment:
This is too complicated just have an inline if 

`None is not opt.copy_attn else ... `